### ISSUE TITLE
Incompatible with android-cordova 10 due to cordova-plugin-browsertab requirement

### ISSUE THREAD

**Issue by jpike88** [firebase/firebase-js-sdk#5801]
https://github.com/firebase/firebase-js-sdk/blob/a777385d67653cdcc3b839149dde867f32b48369/packages/auth/src/platform_cordova/popup_redirect/utils.ts#L250

cordova-plugin-browsertab (last commit in 2017) depends on cordova-plugin-compat, which was also deprecated years ago. When installing android-cordova 10.1.1, cordova-plugin-compat doesn't even install anymore due to version checks. It's an extremely out of date dependency that can't be relied on anymore.

This breaks google-auth method in Cordova.

**Comment by sam-gc** [comment]
Unfortunately, for now there are no plans on moving away from this dependency. It is a required component for performing `signInWithRedirect`.

**Comment by jpike88** [comment]
Ironically, ```cordova-plugin-browsertab``` was made by a Google employee and sits under the Google organisation on GitHub. https://github.com/google/cordova-plugin-browsertab

Closing this is just sweeping the problem under the rug. I'm going to fork the plugin and fix it my end, but considering what a pain it was to figure this out, and for the sake of future developers I've made a PR to make it clear there is an incompatibility.

https://github.com/firebase/firebase-js-sdk/pull/5805
